test/viewmodels/CalculationPageHelperSpec.scala

```diff
@@ -101,6 +105,8 @@ class CalculationPageHelperSpec extends SpecBase with FeatureSwitching {
       penaltyStatus = LPPPenaltyStatusEnum.Accruing,
       penaltyAmountPaid = Some(1001.45),
       penaltyAmountOutstanding = Some(99.99),
+      penaltyAmountPosted = 1001.45,
```

Comment:
This is an accruing penalty, therefore there would be no posted amount. Outstanding amount would be None and paid would also be None